client/src/components/RelatedItemsAndOutfit/Carousel.jsx

```diff
@@ -88,7 +92,11 @@ const Carousel = ({ product, styles, related, reviews, carouselType }) => {
         </div>
         <div
           className="carousel__edge carousel__edge--right"
-          style={{visibility: canScrollRight ? 'visible' : 'hidden', opacity: canScrollRight ? 1 : 0}}
+          style={{
```

Comment:
It's good to see more exception handling in the various levels of data handling. Great work!